packages/eslint-plugin-gestalt/src/helpers/styleValidators.js

```diff
@@ -0,0 +1,104 @@
+// @flow strict
+const colorMap = {
+  '#e60023': `color="red"`,
+  'white': `color="white"`,
+  '#fff': `color="white"`,
+  '#ffffff': `color="white"`,
+  '#333': `color="darkGray"`,
+  '#333333': `color="darkGray"`,
+  '#efefef': `color="lightGray"`,
+  '#767676': `color="gray"`,
+  '#0fa573': `color="green"`,
+  '#0a6955': `color="pine"`,
+  '#364a4c': `color="olive"`,
+  '#0074e8': `color="blue"`,
+  '#004b91': `color="navy"`,
+  '#133a5e': `color="midnight"`,
+  '#b469eb': `color="purple"`,
+  '#8046a5': `color="orchid"`,
+  '#5b2677': `color="eggplant"`,
+  '#6e0f3c': `color="maroon"`,
+  '#f13535': `color="watermelon"`,
+  '#e3780c': `color="orange"`,
+  'transparent': `color="transparent"`,
+  'rgba(51,: 51,51,.8)': `color="transparentDarkGray"`,
+  '#e2e2e2': `color="lightWash"`,
+  '#dadada': `color="darkWash"`,
+};
+
+const borderRadiusMap = {
+  '50%': `rounding="circle"`,
+  '999px': `rounding="circle"`,
+};
+
+const borderMap = {
```

Comment:
I think a regex might be better for this? e.g. it looks like this would miss `lightgray 1px solid`…the order doesn't matter for the three values so I don't think we want to enumerate every possible combination

Or perhaps look for the presence of certain substrings using String.includes

Reply:
Wait…it looks like you are using a regex below for this. I'm confused why this map is needed?

Reply:
This is from Jenny's. I only reorganized the file. The logic hasn't changed.

Reply:
LEAVING AS IS